emcc.py

```diff
@@ -791,13 +792,9 @@ def array_contains_any_of(hay, needles):
       if n in hay:
         return True
 
-  # relaxed-simd implies simd128.
-  if '-mrelaxed-simd' in user_args:
-    user_args += ['-msimd128']
-
   if array_contains_any_of(user_args, SIMD_INTEL_FEATURE_TOWER) or array_contains_any_of(user_args, SIMD_NEON_FLAGS):
-    if '-msimd128' not in user_args:
-      exit_with_error('Passing any of ' + ', '.join(SIMD_INTEL_FEATURE_TOWER + SIMD_NEON_FLAGS) + ' flags also requires passing -msimd128!')
+    if '-msimd128' not in user_args and '-mrelaxed-simd' not in user_args:
```

Comment:
should be `or`?

Reply:
I think to preserve previous semantics it needs to be `and`.   i.e. if they are both not in the argument list.  I could rewrite to be `if not any(x in user_args for x in ('-msimd128', '-mrelaxed-simd')):`.. ?

Reply:
The error message we are shown is only needed with neither flag is present, right?

Reply:
You're right.